conf/script/build-ha-io

```diff
@@ -518,6 +525,16 @@ motr_kernel_rsc_add() {
                                            kind=Optional symmetrical=false
 }
 
+var_motr_rsc_add() {
+    log "${FUNCNAME[0]}: Adding var-motr mounts to Pacemaker..."
```

Comment:
Sorry, I am late for review.

I think we need to delete resource from `prov-ha-reset`

Otherwise sw update will failed

Reply:
I think this was discussed here already - https://github.com/Seagate/cortx-ha/pull/144#discussion_r494450608. But it was a good spot! ;)

Reply:
In prov-ha-reset we delete the entire c1,c2 group so the resource must get deleted as part of that.

Reply:
ohh got it. 
Thanks